Lamdu/Expr/Val/Arbitrary.hs

```diff
@@ -76,8 +76,7 @@ arbitraryLeaf :: GenExpr V.Leaf
 arbitraryLeaf = do
     Env locals globals <- Reader.ask
     join . liftGen . Gen.elements $
-        [ V.LLiteralInteger <$> liftGen arbitrary
```

Comment:
It means it won't generate arbitrary literals for the tests, perhaps weaken them, no? I should have thought of that.